apps/storybook-react/tailwind.config.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
   presets: [designSystemPreset],
   content: [
     '../../packages/design-system-react/src/**/*.{js,jsx,ts,tsx}',
+    '../../packages/design-tokens/stories/**/*.{js,jsx,ts,tsx}',
```

Comment:
I think we can remove this because we aren't currently using tailwind classnames inside of the design token stories

```suggestion
```